src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1267,9 +1311,22 @@ void MarkEntireAssembly (AssemblyDefinition assembly)
 				MarkEntireType (type, includeBaseTypes: false, new DependencyInfo (DependencyKind.TypeInAssembly, assembly), null);
 		}
 
+		void ProcessPerAssemblySteps (AssemblyDefinition assembly)
```

Comment:
I'd not call it Steps it's just old artifact name at this point for the logic

Reply:
Inlined this logic